### DISCUSSION TITLE
Why is v8-compile-cache removed?

### DISCUSSION THREAD

**Discussion by wangjieCode** [webpack/webpack#15958] [Q&A]
I see V8 compile cache in the lock file, but I can't find the reference

**Comment by vankop** [comment]
maybe it is used in some dependencies, webpack doesn't use it

**Reply by alexander-akait** [reply]
ESM support, `v8-compile-cache` doesn't allow to use `import(...)` and it is not fixable

**Reply by wangjieCode** [reply]
webpack uses the commonjs specification， Because you can't use ``import()`` to give up the performance improvement brought by ``v8-compile-cache`` ？

**Reply by alexander-akait** [reply]
Not related to commonjs, it is the bug/limitation on `v8-compile-cache`

**Reply by wangjieCode** [reply]
I still don't quite understand

**Reply by alexander-akait** [reply]
Why you need `v8-compile-cache`?
